src/ContactManager/Contacts.cs

```diff
@@ -0,0 +1,47 @@
+﻿// <copyright file="Contacts.cs" company="PlaceholderCompany">
+// Copyright (c) PlaceholderCompany. All rights reserved.
+// </copyright>
+
+using System.Security.Cryptography.X509Certificates;
+
+namespace ContactManager
+{
+    public class Contacts
+    {
+        public string name;
+        public string email;
+        public string phone;
+        public string additionalNotes;
+        public Contacts(string name, string email, string phone, string additionalNotes)
+        {
+            this.Name = name;
+            this.Email = email;
+            this.Phone = phone;
+            this.AdditionalNotes = additionalNotes;
+        }
+
+        public string Name
+        {
+            get { return name; }
+            set { Name = value; }
+        }
+
+        public string Email
+        {
+            get { return Email; }
```

Comment:
```suggestion
            get { return email; }
```

Reply:
This is recursive. You need to use the variable.

Reply:
Changed as required